ec2/volumes.go

```diff
@@ -186,3 +186,26 @@ func (e *Ec2) ListVolumeSnapshots(ctx context.Context, id string) ([]string, err
 
 	return snapshots, nil
 }
+
+func (e *Ec2) ModifyVolume(ctx context.Context, iops int64, volumeType string, size int64, id string) (*ec2.VolumeModification, error) {
```

Comment:
I think we should just pass `input *ec2.ModifyVolumeInput` here instead of all these parameters, similar to how we do it for `CreateVolume`.
It means you'll need an orchestrator function that you call from the handler.

Reply:
Done.